web/tests/test_views.py

```
@@ -522,31 +522,30 @@ def test_enrolled_student_functionality(self):
-        session_month = self.future_session.start_time.month
-        session_year = self.future_session.start_time.year
-        response = self.client.get(f"{self.detail_url}?month={session_month}&year={session_year}")
+        # Get the future session's month and year
+        session_date = self.future_session.start_time.date()
+        response = self.client.get(f"{self.detail_url}?month={session_date.month}&year={session_date.year}")
 
-        # Test calendar context
-        self.assertTrue("calendar_weeks" in response.context)
-        self.assertTrue("today" in response.context)
+        # Assert response status code
+        self.assertEqual(response.status_code, 200)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)